### ISSUE TITLE
Name class height()  changes!

### ISSUE THREAD

**Issue by duck7000** [duck7000/imdbGraphQLPHP#106]
I changed this method.
Long time ago the output of this method was only available in text form but now it is available as int (Metric only in centimeters)
So i changed this method to output height as metric int|float in centimeters.
And for Imperial i converted the metric centimeter value to feet and inches.

Example:
Height in centimeters: 189.23
Imperial would be this:
feet: 6
inches: 2.50004023

In text form this can be displayed as: 6' 21/2" but other forms are now possible as well

The output array is changed as well like this:
```
(
    [imperial] => Array
        (
            [feet] => 6
            [inches] => 2.50004023
        )
    [metric] => 189.23
)
```

So to sum up

1. Metric is now int|float( in centimeters)
2. imperial is now separated in feet and inches, both as int|float
3. values are not rounded

all info is in the wiki

If there are any issues, bugs or questions let me know here
